packages/http-client-csharp/eng/scripts/Generation.psm1

```diff
@@ -24,7 +24,8 @@ function Get-TspCommand {
     param (
         [string]$specFile,
         [string]$generationDir,
-        [bool]$generateStub = $false
+        [bool]$generateStub = $false,
+        [string[]]$additionalOptions = @()
```

Comment:
nit: rather than making it accept any option, can we use the existing pattern where we have a specific flag/string, e.g. `namespaceOverride`?